spec/controllers/ncr/work_orders_controller_spec.rb

```diff
@@ -61,35 +61,19 @@
   end
 
   describe '#update' do
-    let (:work_order) { create(:ncr_work_order, :with_approvers) }
+    let (:work_order) { create(:ncr_work_order) }
     let (:requester) { work_order.proposal.requester }
     before do
       login_as(requester)
     end
 
-    it "uses existing first approver when there is a blank approver email" do
```

Comment:
this spec is no longer relevant because we are now called `validates :approving_official, presence: true`, so there will be a validation error of a work order is updated with no approving official FK